modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2507,6 +2520,59 @@ public void testGetObjectEntriesWithAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetObjectEntryDocument() throws Exception {
+		_objectEntryLocalService.addObjectEntry(
+			adminUser.getUserId(), 0,
+			_objectDefinition1.getObjectDefinitionId(),
+			HashMapBuilder.<String, Serializable>put(
+				"textObjectFieldName", StringUtil.randomId()
+			).put(
+				"textObjectFieldNameExtension", StringUtil.randomId()
+			).build(),
+			ServiceContextTestUtil.getServiceContext());
+
+		SearchResponse searchResponse = _searcher.search(
+			_searchRequestBuilderFactory.builder(
+			).entryClassNames(
+				_objectDefinition1.getClassName()
+			).emptySearchEnabled(
+				true
```

Comment:
@p-albuquerque can we sort this? `emptySearchEnabled` should be before `entryClassNames`